### PR TITLE
Prevent content from shifting

### PR DESCRIPTION
**Pull request by gleung93**
https://user-images.githubusercontent.com/10873719/200729759-9c39bc83-a4ab-41f4-929d-fb8318038a78.mov

### CODE REVIEW

File: src/components/SharedLayout.js [hunk 1]
```diff
@@ -5,7 +5,9 @@ const SharedLayout = () => {
   return (
     <>
       <NavBar />
-      <Outlet />
+      <div className="container">
```
Comment (gleung93) [inline src/components/SharedLayout.js:8]: Moved this div up from the content components so we can avoid repetition

File: src/index.css [hunk 2]
```diff
@@ -120,17 +121,17 @@ ul {
   display: flex;
 } */
 
-.content {
+/* .content {
```
Comment (gleung93) [inline src/index.css:124]: There were multiple declarations for `.caption` and `.content`. Not sure if that was intentional or not so I commented them out

**Comment by louieknolle** [comment]
Beautiful! Add this to your open source contributions ;)

File: src/index.css [hunk 3]
```diff
@@ -222,11 +223,21 @@ img {
         overflow-x: scroll;
     } */
 
+  .container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 80%;
```
Comment (gleung93) [inline src/index.css:230]: Setting the container to a fixed width is what ultimately resolved the shifting issue. Now this container will stay the same size regardless of the content within

File: src/index.css [hunk 4]
```diff
@@ -239,6 +250,7 @@ img {
 
   .caption {
     margin-left: 4rem;
+    margin-top: auto;
```
Comment (gleung93) [inline src/index.css:253]: Ensure that the caption is always at the bottom of the container

File: src/index.css [hunk 5]
```diff
@@ -16,6 +16,7 @@
 body {
   background-color: var(--white);
   font-family: "Oxygen", sans-serif;
+  overflow: hidden;
```
Comment (gleung93) [inline src/index.css:19]: This prevent the "overscroll" on the page so that only the content moves

**Pull request merged**